test/reducers/TasksReducerTest.js

```diff
@@ -1,4 +1,5 @@
-const assert = require('assert')
+const { assert } = require('chai')
```

Comment:
Switched to chai because it seems like the standard mocha assert framework and it gives us some better assertions.